### PR TITLE
NEW: Front end for posting complete. 

### PR DESCRIPTION
**Pull request by lamaqdahodwala**

### CODE REVIEW

**Comment by mo-balsara** [comment]
I'm tying to link it to the proper JavaScript directory. I'll update the repo so you can see what I'm trying to do.

**Comment by lamaqdahodwala** [comment]
Do you have a specific server open to send requests to?

**Comment by mo-balsara** [comment]
No, how would I do that?

**Comment by mo-balsara** [comment]
Actually yes. Im using the express server in ```index.js```

**Comment by lamaqdahodwala** [comment]
> Actually yes. Im using the express server in `index.js`

Could i have the link
Also, this is the database we would be saving all the posts on right?

**Comment by mo-balsara** [comment]
I haven't yet made a database where all the posts are stored. I thought by server you meant the script that renders the html.

File: src/public/dashboard.html [hunk 1]
```diff
@@ -2,10 +2,22 @@
 <html>
 <head>
   <title>Say What!</title>
-  <link href="views/css/dashboard.css" rel="stylesheet" type="text/css" />
+  <link href="dashboard.css" rel="stylesheet" type="text/css" />
```
Comment (mo-balsara) [inline src/public/dashboard.html]: pretty sure we should keep the css here

File: src/public/dashboard.html [hunk 2]
```diff
@@ -2,10 +2,22 @@
...
 </head>
 <body>
+  <script src="script.js"></script>
```
Comment (mo-balsara) [inline src/public/dashboard.html]: this too

**Review by mo-balsara — COMMENTED** [review]
directory linking still broken :cri:

File: src/public/dashboard.html (hunk 1 above)
Comment (lamaqdahodwala) [reply src/public/dashboard.html]: I fixed the css bro. Just merge my changes.

Comment (mo-balsara) [reply src/public/dashboard.html]: K sure

Comment (mo-balsara) [reply src/public/dashboard.html]: And it works? Like you tested it out

**Comment by mo-balsara** [comment]
Wait where is the merge button? Lol.

**Review by mo-balsara — APPROVED** [review]
Pog

**Comment by mo-balsara** [comment]
Congrats on contributing if you wanna help more just ask. Also the css and JavaScript doesnt work

File: src/public/dashboard.html (hunk 1 above)
Comment (lamaqdahodwala) [reply src/public/dashboard.html]: yes. merge them changes and all the css and js words.

**Comment by lamaqdahodwala** [comment]
Dude. Its probably because I was rendering the html directly. Don't make a whole server for rendering it. Repl does it itself.

**Comment by mo-balsara** [comment]
I usually make an _express_ server in the ```index.js``` file because in the file you can attach different packages, so if you could find a way to have it work from rendering the html from the _express_ server, that would help. **I'm giving you edit access for now for this.**

**Comment by mo-balsara** [comment]
I'll also invite you to the replit side.

**Pull request merged**